src/Server/Program.fs

```diff
@@ -65,16 +44,16 @@ let main args =
             |> Option.defaultValue Database.DatabaseType.FileSystem
 
         let port = getPortsOrDefault 8085us
-        
-        WebHost
-            .CreateDefaultBuilder()
-            .UseWebRoot(clientPath)
-            .UseContentRoot(clientPath)
-            .ConfigureLogging(configureLogging)
-            .Configure(Action<IApplicationBuilder> (configureApp database clientPath))
-            .UseUrls("http://0.0.0.0:" + port.ToString() + "/")
-            .Build()
-            .Run()
+
+        let app = application {
+            router (WebServer.webApp database)
+            url ("http://0.0.0.0:" + port.ToString() + "/")
+
+            use_jwt_authentication JsonWebToken.secret JsonWebToken.issuer
+            use_static clientPath
+            use_gzip
+        }
+        run app
```

Comment:
@Krzysztof-Cieslak I can see most of the mappings from before -> after, but do the calls for e.g. `CreateDefaultBuilder()` etc. simply get put in for free within the `application { }` CE?

Reply:
Yes, Satur tries to hide some of this additional fluff away - so `CreateDefaultBuilder` and `Build` are provided automatically by Saturn. `UseWebRoot` and `UseContentRoot` are set inside `use_static` helper. 

And `run app` function is just `app.Run()` call